Rise-Vision/common-header
# Checking for element existence before adding listener

## Description
Some modals don't have modal content. This is causing an error to appear sometimes:
`TypeError: Cannot read property 'addEventListener' of null`
Checking for element existence before adding listener

## Motivation and Context
Fix uncaught error

## How Has This Been Tested?
To validate, go to Storage or open a Template in Template Editor (that has an Image component). Open a modal such as the Select Company modal. Look at the console, the error should appear.

Fix staged at https://apps-stage-2.risevision.com

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 

**Manual Test**  Tested to ensure error doesn't occur
**Automated Test** No need to update
**Monitoring** Will validate the release once in Production
**Rollback** Rollback can be done by reverting the PR and following the deployment process
**Documentation** Error catcher added, no need to update documentation
**Internal Communication** Will notify interested parties of the fix being deployed

- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why
